### DISCUSSION TITLE
Any idea how to remove this padding?

### DISCUSSION THREAD

**Discussion by Tobshub** [neovim/neovim#29756] [Q&A]
![image](https://github.com/user-attachments/assets/b4018e78-39c3-4026-88d2-5387c9c0023c)
In my hover window there's this padding that annoys the heck out of me. I just want it gone, but I have no idea how. I've checked the help for vim.lsp.handlers.hover (and all the places it redirects you to) but nothing says anything about the padding or anything related.
To clarify, what I mean by padding is the space above and below the actual text in the hover window. I'd like it as tight as possible

**Comment by clason** [comment]
This is not padding but concealed Markdown codeblocks. There's currently no way of removing it until the "anticonceal" PR is finished (which will take a while).
